<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOSTypeDialog</class>
 <widget class="QDialog" name="AddOSTypeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый тип ОС</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Тип ОС</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButtonWin10">
          <property name="text">
           <string>Win 10</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButtonWinXP">
          <property name="text">
           <string>Win XP</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButtonWin7">
          <property name="text">
           <string>Win 7</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Embeded</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Разрядность</string>
     </property>
     <widget class="QRadioButton" name="radioButton64">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>102</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>64 bit</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton32">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>102</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>32 bit</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
